python_transport/wirepas_gateway/protocol/mqtt_wrapper.py

```diff
@@ -241,6 +261,7 @@ def run(self):
             # As this thread is daemonized, inform the parent that this
             # thread has exited
             self.on_termination_cb()
+            exit(2)
```

Comment:
There is no semantic at the moment on exit code but at least it is not homogeneous. Above we call exit(-1), and here exit(2). So either both negative or both positive.

Reply:
But does it make sense to call exit() here?
termination_cb is already exiting after stopping the dbus client.

So it would be more clean to exit from transport_service instead.
But the other exit in this file should be removed too, so I'll move all of them at the same time later

Reply:
I will remove it. I guess it's a workaround for the dbus client issue, which is fixed now.